### ISSUE TITLE
PASU registration: studied and sorted codes getting combined

### ISSUE THREAD

**Issue by luizaogs** [arch-kiosk/arch-kiosk-office#2167]
Maybe this is what it's supposed to look like but it confused me, so I'm bringing it up. PASU has the two following registration codes: 

S = sort(ed) 
St = to study/studied

Both start with S, which seems to be causing problems. If I click on S for sorted in the checkbox in registration, I get the S as expected in the todo column. But then if I also check St for studied in the to do checkbox for that same record, I end up with only "St" in the to do column. Which is confusing, because then it looks like I only need to study the thing, not sort and study it.

![IMG_0349](https://github.com/arch-kiosk/arch-kiosk-office/assets/67970402/c9310610-fecf-44e8-9313-c501bebeebb7)

![IMG_0350](https://github.com/arch-kiosk/arch-kiosk-office/assets/67970402/7c884f3f-9801-47d4-9096-df26dabb06a3)

**Comment by urapadmin** [comment]
this is simply not allowed anymore. PASU used a bunch of really funny registration codes.
Registration codes must be one character wide and only the exclamation mark is allowed as a second modified to a symbol. 
We have solved this for PASU by turning all "St" codes to "T". see #2165

**Comment by luizaogs** [comment]
The issue mentioned in #2165 about automatic codes getting assigned to new CM has been resolved. 

Still something weird with the Ss and Ts, however. It seems like what should be Ts are ts? Which means that if one clicks on the S and T in the to do column one still ends up with an St, which (to me at least) seems confusing. Can you make the old Sts into capital Ts?

Perhaps more important, though: 
If I click on to be sorted (S) in the to do column, I get an S. Great. 
If I then click on to be studied (T) in the do to column, I get an St (with a lower case t, as I mentioned above). Weird, but fine.
Here is when things become weirder: 
I then click on the T in the done column, expecting to only get the T. But when I click out of the checkboxes dialog, I see an St in the done column instead. I click on the checkbox dialog again, and now both to be sorted and to be studied are marked as done. Is that on purpose?

**Comment by urapadmin** [comment]
Those are still configuration issues.

**Comment by urapadmin** [comment]
yes, and my kitchen statement that registration might be more complicated than it looks is also not true anymore. Meanwhile it is really only the registration_todo and registration_done keys that need to be configured. And there was simply still an St= lurking in one of them.

Should be done now (no new software version needed)

**Comment by luizaogs** [comment]
Sorry, I think I'm just confused at this point. Should clicking on "to be studied" also automatically check "to be sorted"? Because that's still happening. So I'm still getting the St.

**Comment by urapadmin** [comment]
no, it shouldn't and it doesn't in the version I used to change that part of configuration. Ít appears that my configuration change did not make it. Did I not synchronize and just reset the workstations? I don't know. 
I have repeated the config and now I am sure I have synchronized. Please try again.

**Comment by luizaogs** [comment]
Looks good now.
